<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OffLine_TestClass</class>
 <widget class="QMainWindow" name="OffLine_TestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1416</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OffLine_Test</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="2,1,1">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
      <item>
       <widget class="QLabel" name="label_Show">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="label_3"/>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>图像列表：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="lw_ImageList"/>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="Button_Start">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>开始</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="checkBox_OK">
            <property name="text">
             <string>OK停止</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_NG">
            <property name="text">
             <string>NG停止</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="Button_CameraSet">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>设置</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="Button_Exit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;Arial&quot;;</string>
          </property>
          <property name="text">
           <string>退出</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>检测结果：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget_Result"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1416</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="OffLine_Test.qrc"/>
 </resources>
 <connections/>
</ui>
